### PR TITLE
Use message type instead of prototype

### PR DESCRIPTION
**Pull request by 0x26res**
I've fixed the issue and added some tests. 

But I wasn't reproduce the issue with this repo set up. The issue stems from the fact that in some environment:
```
MessageFactory().GetPrototype(message_type.DESCRIPTOR) is not messsage_type
```

I think it must have something to do with a different combination of `protoc` and `protobuf` python library.

Does anyone happen to know what version of `protoc` has been used to generate the pb2.py files?

Thanks.

### CODE REVIEW

File: tests/integration/schema_registry/test_proto_serializers.py
Comment (pranavrth) [inline tests/integration/schema_registry/test_proto_serializers.py:26]: It's better to keep data in their respective test files. Its fine here to replicate data in each file if we require that.

File: tests/schema_registry/test_proto.py
Comment (pranavrth) [inline tests/schema_registry/test_proto.py:35]: No need for such complex data while testing our usecase. Keep only what is really required.

Move data near the testcase.

File: tests/integration/schema_registry/test_proto_serializers.py
Comment (pranavrth) [inline tests/integration/schema_registry/test_proto_serializers.py:120]: Keep this extra line. Each test is separated by 2 lines.

File: src/confluent_kafka/schema_registry/protobuf.py
Comment (pranavrth) [inline src/confluent_kafka/schema_registry/protobuf.py:514]: Remove this extra line.

File: tests/schema_registry/test_proto.py
Comment (pranavrth) [inline tests/schema_registry/test_proto.py:117]: Please write a better name for the test which suggests what we are trying to test. Something in the line of serializing, deserializing and then again serializing. Better to have verbose test name.

File: tests/schema_registry/test_proto.py
Comment (pranavrth) [inline tests/schema_registry/test_proto.py:129]: Keep all asserts continuously.

Add a line break before assert.

File: tests/schema_registry/test_proto.py
Comment (pranavrth) [inline tests/schema_registry/test_proto.py:135]: We are removing this code altogether, so I think we don't need to test this. Please remove this function.

**Review by pranavrth — CHANGES_REQUESTED** [review]
Please fix the build issues. Mostly related to the styling. 

https://app.travis-ci.com/github/confluentinc/confluent-kafka-python/builds/254834885#L827

**Comment by 0x26res** [comment]
@pranavrth thanks for the feed back. I've addressed the comments.

The CI is failing, but (unless I'm mistaken) I think the serdes branch has been left in a bad state, some tests need fixing.

**Comment by pranavrth** [comment]
Hello @0x26res,

The testcases related to protobuf were failing in **serdes** branch. I have fixed and merged the test cases. Please resolve the conflict and it should work fine.

**Comment by 0x26res** [comment]
@pranavrth thanks for fixing, it looks good now.

**Comment by pranavrth** [comment]
LGTM

**Review by pranavrth — APPROVED** [review]

**Comment by 0x26res** [comment]
@pranavrth I don't have write access so I won't be able to merge. Can I leave this to you? Thanks

**Comment by 0x26res** [comment]
@mhowlett is that not going through? Should I send another MR?

**Pull request closed without merging**
